lib/code_corps/github/event/pull_request/pull_request.ex

```diff
@@ -33,12 +32,12 @@ defmodule CodeCorps.GitHub.Event.PullRequest do
     end
   end
 
-  @spec validate_payload(map) :: {:ok, :valid}
-                               | {:error, :unexpected_payload}
+  @spec validate_payload(map) :: {:ok, :valid} | {:error, :unexpected_payload}
```

Comment:
I think the old spec is maybe how the formatter is doing it now?

Reply:
Oh possibly. It was inconsistent across the handlers, so I just normalized them by hand. I can change it back if you want, but considering the discussion note in the PR, I feel like this code could be moved to the caller at some point soon. Your call, though.